src/niweb/apps/noclook/forms/common.py

```diff
@@ -47,6 +47,49 @@ def get_node_type_tuples(node_type):
     choices.extend([tuple([item['handle_id'], item['name']]) for item in names])
     return choices
 
+class TagListField(forms.CharField):
+    def __init__(self, *args, **kwargs):
+        if 'widget' not in kwargs:
+            kwargs['widget'] = forms.Textarea(attrs={'cols': '120', 'rows': '2'})
+        super(TagListField, self).__init__(*args, **kwargs)
+
+    def clean(self, value):
+        """
+        Validate and clean the input, return list of tags.
+        """
+        value = super(TagListField, self).clean(value)
+        result = []
+        errors = []
+
+        # Split by comma
+        for tag in StringIO(value).read().split(','):
```

Comment:
So the field is comma seperated, guess that works. Not sure if it would have been just as fine to just do space separated.